sw/airborne/modules/computer_vision/cv.c

```diff
@@ -113,6 +113,13 @@ int8_t cv_async_function(struct cv_async *async, struct image_t *img)
   if (async->img_copy.buf_size == 0) {
     image_create(&async->img_copy, img->w, img->h, img->type);
   }
+#if CV_ALLOW_VIDEO_TO_CHANGE_SIZE
+  // Note: must be enabled explicitly as not all modules may support this. (See issue #2187)
+  if (img->buf_size > async->img_copy.buf_size) {
```

Comment:
only grow?

Reply:
Yes, as suggested by @kirkscheper  [here](https://github.com/paparazzi/paparazzi/issues/2187#issuecomment-345455394). It prevents some unnecessary free'ing and alloc'ing at the cost of a larger memory consumption (that was already required to hold the larger frame anyway).